### PR TITLE
fix: do not print generic error message for expected errors

### LINKED ISSUE #21: Don't print generic error message for expected errors

### PR DESCRIPTION
**Pull request by km1chno**
Resolves #21 

## Before 

Zrzut ekranu 2024-09-19 o 12 16 03

## After 

Zrzut ekranu 2024-09-19 o 12 15 48

### CODE REVIEW

File: src/extensions/interactive.ts [hunk 1]
```diff
@@ -123,7 +123,7 @@ module.exports = (toolbox: CycliToolbox) => {
     }).prompt()
 
     if (isCancel(confirmed)) {
-      throw Error('The script execution has been canceled by the user.')
+      throw CycliError('The script execution has been canceled by the user.')
```
Comment (maciekstosio) [inline src/extensions/interactive.ts:126]: Maybe we should also have some predefined errors so we don't repeat this every time? Like const CycliErrorCanceled = new CycliError(...), wdyt?

Comment (maciekstosio) [reply src/extensions/interactive.ts:126]: It could be even just here at the top of file as it seems to be just in this file.

**Review by maciekstosio — APPROVED** [review]
One nit pick and we are good to go!

File: src/extensions/interactive.ts (hunk 1 above)
Comment (km1chno) [reply src/extensions/interactive.ts]: good idea, done in 8cc3ad70823b34731eae1f07ddab2e262ac195e9

**Review by maciekstosio — APPROVED** [review]

**Pull request merged**